Modules/remote-bundles.json

```diff
@@ -0,0 +1,119 @@
+{
+"Windows-x86_64" : [
+	{"url" : "https://repo.jasp-stats.org/bundles/Windows-x86_64/jaspBFF.JASPModule",                     "checksum" : "30ef8fcc0a4f17f6d7e4540fd7c3f234f9879533aec17ab75a3672b6f83aa12e"},
```

Comment:
So these ".JASPModule" files contain the entire registry of all version of that module and each file used in each?

Reply:
Ill read the pdf later btw, might explain this

Reply:
They can but specially for your shitty internet they can also just contain a manifest that describes how to obtain them :p

Reply:
Amazing haha
Should keep me happy until august... :')